backend/app/routers/authentication/authentication.py

```
@@ -16,6 +16,7 @@
 from app.utils.constants import EMAIL
 
 from .authenticate import (
+    generate_verification_code,
```

Ruff (0.8.2)

`.authenticate.generate_verification_code` imported but unused

Remove unused import: `.authenticate.generate_verification_code`

(F401)